wordpress-mobile/WordPress-iOS
# Move Fastlane to Project Root

Aligns this project with others to put the `fastlane` directory at the root of the project.

**To Test:**
Note that the following CI jobs are passing for this PR:
- Build Tests
- Unit Tests
- UI Tests (both)
- Installable Build
- Translation Build (note that the build in appetize.io is the one associated with the CI run for this PR)

**To Review:**
This is (necessarily) a _huge_ PR because it moves every file in the `fastlane` directory. I'd recommend reviewing this by commit – the first just moves all of the files into the new location – subsequent ones alter them to make them compatible with the new location.

---

It's possible that there might be other breaking changes I haven't encountered while testing, but I'd like to land this in the release branch (then in `develop`) as soon as possible in order to avoid conflicts with other changes – I can fix other broken tooling if/when I run into it – this should be the vast majority of it.

--

Reviews: I'd like review from both @AliSoftware and @loremattei for this – @loremattei because he's most familiar with all of it and @AliSoftware because I streamlined a bunch of the pathing and variables and I'd love your input on the changes.

Thank you!!
